db/migrate/20170615122507_remove_columns_in_mail_templates.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveColumnsInMailTemplates  < ActiveRecord::Migration
```

Comment:
This is only neded if we create the columns in the first place.

It's needed on you computer only, so I suggest, in the Branch, remove the columsn from the previous migration-file, and then remove this whole file. It makes the filesystem a bit cleaner.